web-fixtures/sub-folder/cookie_page4.php

```diff
@@ -0,0 +1,18 @@
+<?php
+if (!isset($cookieValue)) {
+    $cookieValue = 'srv_var_is_set';
+}
+
+$cookiepath = dirname($_SERVER['REQUEST_URI']) . '/';
```

Comment:
rename `$cookiepath` into `$cookiePath`

---

not according to a coding standard